### ISSUE TITLE
ModuleNotFoundError: No module named 'ruamel'

### ISSUE THREAD

**Issue by m4rkl1u** [microsoft/nni#3276]
**Environment**:
- NNI version: 1.9.0
- NNI mode (local|remote|pai): Local
- Client OS: Ubuntu
- Server OS (for remote mode only): Local
- Python version: 3.6.0
- PyTorch/TensorFlow version: tensorflow==2.3.0
- Is conda/virtualenv/venv used?: conda
- Is running in Docker?:Yes

**Log message**:

```
   /opt/conda/lib/python3.6/site-packages/nni/package_utils.py in <module>
          8 from pathlib import Path
          9 import importlib
    ---> 10 import ruamel.yaml as yaml
         11 
         12 from .constants import BuiltinAlgorithms
    
    ModuleNotFoundError: No module named 'ruamel'
 
```

**What issue meet, what's expected?**:
It seems the conda install ruamel is ruamel_yaml, I expect that if the nni running in docker+conda environment could identify the ruamel path

**How to reproduce it?**: 

**Additional information**:

**Comment by liuanhua110** [comment]
change the 'import ruamel.yaml as yaml' to 'import ruamel_yaml as yaml' it works for me

**Comment by kvartet** [comment]
> change the 'import ruamel.yaml as yaml' to 'import ruamel_yaml as yaml' it works for me

Thanks for your solution, maybe @m4rkl1u can try his method.

**Comment by kvartet** [comment]
met this error in Windows :(

**Comment by potoo0** [comment]
- windows 10
- python 3.8
 could install `ruamel.yaml` to solve this:
```
pip install --ignore-installed ruamel.yaml
```

**Comment by kvartet** [comment]
To avoid this issue, NNI substitutes ``ruamel.yaml`` with PyYAML in pr #3702, which will be released in v2.3.
